monai/apps/auto3dseg/auto_runner.py

```diff
@@ -189,6 +195,7 @@ def __init__(
         hpo_backend: str = "nni",
         ensemble: bool = True,
         not_use_cache: bool = False,
+        templates_path_or_url=None,
```

Comment:
It can be useful to add the type hint here. Thanks!

Reply:
okay